logger.go

```diff
@@ -57,7 +59,7 @@ func (sumoLogger *sumoLogger) consumeLogsFromFile() {
   var log logdriver.LogEntry
   for {
     if err := dec.ReadMsg(&log); err != nil {
-      if err == io.EOF {
+      if err == io.EOF || err == os.ErrClosed || strings.Contains(err.Error(), "file already closed") {
```

Comment:
Unfortunately I agree, but I don't know how I could improve this, unless I could somehow reproduce this situation locally after having added a log line to get the error code. I think this could be possible if I close the container while the logging driver is still reading lots of data? But I currently don't have the bandwidth to play with this idea.

I also couldn't find definitive answers regarding the error code by searching online 😞 

FWIW, the error code could also change, and some other docker logging driver plugins are currently using this same string check. Maybe this is fine as a temporary fix, but could be revisited? What do you think?